Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/util/wrappers/ColorPickerViewInteraction.java

```diff
@@ -55,12 +56,21 @@ public ViewInteraction onOkButton() {
 		return onView(withId(R.id.color_chooser_button_ok));
 	}
 
+	public ViewInteraction onCancelButton() {
```

Comment:
Thank you for extending the wrappers, I think this improves our test readability quite a lot.